### ISSUE TITLE
training 

### ISSUE THREAD

**Issue by HeChengHui** [falibabaei/yamnet_finetun#1]
@falibabaei 
Thank you for the finetuning script. 

i am currently trying to train on 2k+ 0.96s long clips with 3 classes. Am i supposed to change `yamnet=yamnet_model.yamnet_frames_model_transfer(params,1)` -> `yamnet=yamnet_model.yamnet_frames_model_transfer(params,3)`?

would softmax still be the correct activation function?

**Comment by falibabaei** [comment]
Hi,

Yes, the second argument is the number of classes that you have: `yamnet_frames_model_transfer(params, num_classes)`. If you look at `yamnet.py`, the `num_classes` parameter is used to adjust the output of the `softmax` function.

**Comment by HeChengHui** [comment]
@falibabaei 
Thank you for the confirmation. If i want to use the trained model, do you know which part should i edit in the original yamnet file to adjust the last layer?
